hailjwt/test/test_jwt.py

```diff
@@ -52,3 +52,9 @@ def test_modified_cyphertext_is_error():
 
 def test_get_domain():
     assert hj.get_domain('notebook.hail.is') == 'hail.is'
+
+
+def test_unsafe_decode():
+    c = hj.JWTClient(hj.JWTClient.generate_key())
```

Comment:
I think JWTClient should make the secret optional in its constructor. This secret won't be useful